backend/routes/movies.js

```diff
@@ -1,8 +1,9 @@
 const express = require('express');
+const MoviesController = require('../controllers/moviesController');
 const router = express.Router();
 
-router.get('/', (req, res) => {
-  res.json({ message: 'Hello from movies api!' });
+router.get('/', MoviesController.getByFilters, (req, res) => {
```

Comment:
you need to pass a param in your get request ex. router.get('/:genreId', moviesController ....)

Reply:
rewrote route to /:genreId, i can add more params if needed